faq/capabilities.rst

```diff
@@ -0,0 +1,222 @@
+====================
+What is a Capability
```

Comment:
Suggest making this question more precise - 
What is a capability? (long answer)

and link to this from the other page
What is a capability? (short answer) - page at faq/what_is_a_capability.rst